django-backend/fecfiler/transactions/schedule_a/serializers.py

```diff
@@ -144,7 +150,19 @@ def update(self, instance, validated_data: dict):
                 if attr != "id":
                     setattr(instance.schedule_a, attr, value)
             instance.schedule_a.save()
-            return super().update(instance, validated_data)
+            updated = super().update(instance, validated_data)
+            self.execute_update_future_transaction_contacts_task_if_needed(
+                validated_data.get('contact_id'), schedule_a_data
+            )
+            return updated
+
+    def execute_update_future_transaction_contacts_task_if_needed(
```

Comment:
What if instead of going to each transaction and juggling whether or not it's been updated then firing off a new task, can we just on contact update, find/query all subsequent transactions and call update on that queryset.